### ISSUE TITLE
A keyboard shortcut to restart a game

### ISSUE THREAD

**Issue by Crinibus** [Bollos00/LibreMines#22]
It would really be nice to have a keyboard shortcut to restart a game either only after a win/lose or also mid-game. This would make it so you don't have to use the mouse to restart a game when playing in keyboard controller mode

**Comment by Bollos00** [comment]
You should de able to navigate through the buttons of the interface with **Tab** and **Shift+Tab**.

During a game (or after end one), try pressing **Tab** and then, when the Restart button gets highlighted, press Space (or Enter). That way you will probably be able to restart a game without the mouse.

I can also add **Ctrl+R** shortcut to make this process easy on the future.

**Comment by Crinibus** [comment]
Oh yeah, I see. Though the Ctrl+R shortcut sounds great and would be nice to have.

Btw it's a really nice game.

**Comment by Bollos00** [comment]
I will add the shortcut whenever I get time, then I let you know.


> Btw it's a really nice game and I love that you can use the keyboard to play.

Thank you 🙂.

**Comment by Bollos00** [comment]
@Crinibus I just made a new release and added the shortcut.

If everything is okay you can close this issue.

**Comment by Crinibus** [comment]
Awesome! Thank you :)
